packages/mermaid/src/diagrams/pie/pieRenderer.ts

```diff
@@ -119,6 +119,7 @@ export const draw: DrawDefinition = (text, id, _version, diagObj) => {
       return ((datum.data.value / sum) * 100).toFixed(0) + '%';
     })
     .attr('transform', (datum: d3.PieArcDatum<D3Section>): string => {
+      // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
       return 'translate(' + labelArcGenerator.centroid(datum) + ')';
```

Comment:
Should we switch this to a template string instead?

```suggestion
      return `translate(${labelArcGenerator.centroid(datum)})`;
```

Reply:
The problem is that the `.centroid` function returns `[number, number]`. So we should look into this properly, maybe when it creates an issue.